getting-started/your-first-feature/7_running_the_new_feature.md

```diff
@@ -0,0 +1,30 @@
+It's time to run the app with the newly added feature!
+
+In the root create-daml-app folder of terminal 1 run
+
+```
+daml start
+```{{execute T1}}
+
+In terminal 2 run
+
+```
+yarn start
+```{{execute T2}}
+
+which will start the UI. As a reminder, this starts the web UI connected to the running Sandbox and JSON API server. Once the web UI has been compiled and started, you should see `Compiled successfully!` in your terminal. You can now [open the UI tab](https://[[HOST_SUBDOMAIN]]-3000-[[KATACODA_HOST]].environments.katacoda.com), where you should see the same login page as before
+
+![Login Screen](/nemanja/courses/getting-started/your-first-feature/assets/create-daml-app-login-screen.png)
+
+Once you’ve logged in, you’ll see a familiar UI but with our new Messages panel at the bottom!
+
+![Messaging Feature](/nemanja/courses/getting-started/your-first-feature/assets/create-daml-app-messaging-feature.png)
+
+Go ahead and add follow more users, and log in as some of those users by [right clikcing on this link and opening a separate tab in your browser](https://[[HOST_SUBDOMAIN]]-3000-[[KATACODA_HOST]].environments.katacoda.com) to follow yourself back. Then, if you click on the dropdown menu in the Messages panel, you’ll be able to see some followers to message!
```

Comment:
`s/add follow more/follow some/`

Reply:
I'd take that sentence apart: Go ahead and follow some users. Then, log in as another user by ...